pkg/api/v1alpha1/nutanixmachineconfig.go

```diff
@@ -57,7 +57,7 @@ type NutanixMachineConfigGenerateOpt func(config *NutanixMachineConfigGenerate)
 // NewNutanixMachineConfigGenerate returns a new instance of NutanixMachineConfigGenerate
 // used for generating yaml for generate clusterconfig command.
 func NewNutanixMachineConfigGenerate(name string, opts ...NutanixMachineConfigGenerateOpt) *NutanixMachineConfigGenerate {
-	emptyString := ""
+	enterNameString := "<Enter name here>"
```

Comment:
Can you make this a formatted string
```suggestion
	enterNameString := "<Enter %s name here>"
```
and then do 
`fmt.Sprintf(enterNameString, "image")`
`fmt.Sprintf(enterNameString, "Prism Element cluster")`
`fmt.Sprintf(enterNameString, "subnet")`
 below where it's used

Reply:
added following 
<pre>
			Image:          NutanixResourceIdentifier{Type: NutanixIdentifierName, Name: func() *string { s := fmt.Sprintf(enterNameString, "image"); return &s }()},
			Cluster:        NutanixResourceIdentifier{Type: NutanixIdentifierName, Name: func() *string { s := fmt.Sprintf(enterNameString, "Prism Element cluster"); return &s }()},
			Subnet:         NutanixResourceIdentifier{Type: NutanixIdentifierName, Name: func() *string { s := fmt.Sprintf(enterNameString, "subnet"); return &s }()},

</pre>

Reply:
added